lib/all_futures.rb

```diff
@@ -2,12 +2,15 @@
 
 require "active_entity/railtie"
 require "active_record/errors"
+require "active_record/integration"
+require "active_record/validations"
```

Comment:
What is `AR:Validations` being used for?

AE already implements its own validation system:

https://github.com/jasl/activeentity/blob/dc2c91cdf7d2e248aefffe201082de565e63feca/lib/active_entity/validations.rb

I don't know what impact this might have on the existing stuff. Is it necessary?

Reply:
Good spot!